make/perf.mk

```
@@ -22,5 +22,5 @@ perf-run:
 	oc get toolchaincluster -n $(MEMBER_NS)
 	oc get toolchaincluster -n ${MEMBER_NS_2}
 	-oc new-project $(TEST_NS) --display-name perf-tests 1>/dev/null
-	ARTIFACT_DIR=${ARTIFACT_DIR} USER_COUNT=3000 USER_BATCH_SIZE=100 MEMBER_NS=${MEMBER_NS} MEMBER_NS_2=${MEMBER_NS_2} HOST_NS=${HOST_NS} REGISTRATION_SERVICE_NS=${REGISTRATION_SERVICE_NS} operator-sdk test local ./test/perf --no-setup --operator-namespace $(TEST_NS) --verbose --go-test-flags "-timeout=120m -failfast" || \
+	ARTIFACT_DIR=${ARTIFACT_DIR} USER_COUNT=3000 USER_BATCH_SIZE=100 MEMBER_NS=${MEMBER_NS} MEMBER_NS_2=${MEMBER_NS_2} HOST_NS=${HOST_NS} REGISTRATION_SERVICE_NS=${REGISTRATION_SERVICE_NS} go test ./test/e2e -v -timeout=60m -failfast || \
```

shouldn't we run the tests in `./test/perf` ?

---

yeah, right :man_facepalming: thanks
```suggestion
	ARTIFACT_DIR=${ARTIFACT_DIR} USER_COUNT=3000 USER_BATCH_SIZE=100 MEMBER_NS=${MEMBER_NS} MEMBER_NS_2=${MEMBER_NS_2} HOST_NS=${HOST_NS} REGISTRATION_SERVICE_NS=${REGISTRATION_SERVICE_NS} go test ./test/perf -v -timeout=60m -failfast || \
```